src/DefaultFormatter.php

```diff
@@ -80,7 +80,7 @@ public function format(\stdClass $tweet)
             '%in_reply_to_screen_name%' => $tweet->in_reply_to_screen_name,
             '%in_reply_to_status_id_str%' => $tweet->in_reply_to_status_id_str,
             '%in_reply_to_user_id_str%' => $tweet->in_reply_to_user_id_str,
-            '%place%' => $tweet->place,
+            '%place%' => $tweet->place->full_name,
```

Comment:
Indeed. If it's null there is a Notice generated, but it won't crash. There are other places that can have null values as well. The YouTube plugin generates a fair number of notices for example :p

I'm not sure of a 'pretty' way to check for null, especially since it will need to be done in many places. Any suggestions?